core/src/main/resources/jenkins/security/ApiTokenProperty/config_pt_BR.properties

```diff
@@ -0,0 +1,47 @@
+# The MIT License
+#
+# Copyright (c) 2004-, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+Show\ Legacy\ API\ Token=Mostrar\ Passe\ De\ API\ Legado
+TokenDisplayedOnce=Copie este passe agora porque ele n\u00E3o poder\u00E1 ser recuperado no futuro.
+RevokedToken=O passe legado foi revogado
+CurrentTokens=Passe(s) atual(is)
+LegacyToken=Recomendamos fortemente que voc\u00EA revogue este passe legado e o troque por um novo rec\u00E9m gerado para aumentar a seguran\u00E7a.
+ConfirmRevokeSingle=Voc\u00EA tem certeza de que quer revogar este passe? Aplica\u00E7\u00F5es que o estejam usando n\u00E3o v\u00E3o conseguir se conectar mais.
+Cancel=Cancelar
+StatisticsDisabled.Title=Estat\u00EDsticas de use de passe est\u00E3o atualmente desabilitadas
+TokenCreation=Criado {0} dia(s) atr\u00E1s
+Legacy\ API\ Token=Passe\ de\ API\ legado
+TokenNeverUsed.Title=Recomendamos fortemente que voc\u00EA revogue os passes que voc\u00EA n\u00E3o planeja utilizar
+StatisticsDisabled=Nenhuma estat\u00EDstica dispon\u00EDvel
+TokenNeverUsedTooltip=---TranslateMe 698--- There is no last use date for that token
```

Comment:
is this intentional?

Reply:
Absolutely not, I just fixed and push again.